### ISSUE TITLE
PayPal Plus - User Reported Issues

### ISSUE THREAD

**Issue by angelleye** [angelleye/paypal-woocommerce#450]
https://wordpress.org/support/topic/paypal-plus-lastschrift-payment-method-not-working?replies=9#post-8300517

**Comment by dagoebel** [comment]
Hi, 

we talked about this issue on PayPal yesterday, and I recorded a video for you, so you can understand it better...

https://www.youtube.com/watch?v=C4KmV4T0yao&feature=youtu.be

Thank you,
Daniel Goebel

**Comment by angelleye** [comment]
Ok, thanks.

Unfortunately, PayPal Plus isn't very high on our priority list right now because PayPal isn't supporting us with it the way they do the other products.

What we're probably going to end up doing is split this out into its own plugin, and then we'll focus on making everything solid with it at that point.

It's probably going to be a couple of months before we can focus on that, though, unfortunately.  I'm sorry I don't have better immediate news for you.
